Controller/ConnectController.php

```diff
@@ -237,7 +237,7 @@ public function connectServiceAction(Request $request, $service)
             ? $this->createForm(FormType::class)
             : $this->createForm('form');
         // Handle the form
-        $form->handleRequest($request);
+        $form->submit($request->request->all());
```

Comment:
This should be just `$request`.

Reply:
No that will also not work, the only thing that I got working was the statement above!

Reply:
Strange, please share more details about how to reproduce the issue.

Reply:
The form has name "form", but the form doesn't hold data so the form doesn't get submitted.

HttpFoundationReuqestHandler has the check if the data has something with the form name:

`
$request->request->has($name) || $request->files->has($name)
`

But the form doesn't really have data so the POST data is empty.

Reply:
The alternate solution is to create a named form without a name, that fixed the issue as well.

Btw I'm symfony 3.4

Reply:
@stloyd can you check which name your form has?